subsys/bluetooth/controller/ll_sw/isoal.h

```diff
@@ -8,6 +8,11 @@
 #include <zephyr/types.h>
 #include <zephyr/toolchain.h>
 
+#if defined(CONFIG_BT_CTLR_ISO_RX_SDU_BUFFERS) && (CONFIG_BT_CTLR_ISO_RX_SDU_BUFFERS > 0)
+#define ISOAL_CONFIG_BUFFER_RX_SDUS_ENABLE
```

Comment:
`XXX_CONFIG_YYY` seems like a poor name, as it can be mistaken for a Kconfig with a typo.